addons/website_event_track/static/src/js/website_event_track.js

```diff
@@ -96,3 +96,16 @@ publicWidget.registry.websiteEventTrack = publicWidget.Widget.extend({
         }
     },
 });
+
+publicWidget.registry.eventTrackForm = publicWidget.Widget.extend({
```

Comment:
I think it's better to declare this widget in a new file called something like `event_track_search.js` to match the `event_sponsor_search.js` one :slightly_smiling_face: 
and maybe renaming the widget to `websiteEventSearchTrack` to be more explicit?